pubnub-kotlin/pubnub-kotlin-api/src/appleMain/kotlin/com/pubnub/kmp/factories.ios.kt

```diff
@@ -310,7 +312,7 @@ private fun mapAppContextEvent(from: KMPAppContextEventResult?): PNObjectEventMe
                     eTag = from.metadata().eTag().orEmpty(),
                     updated = from.metadata().updated().orEmpty(),
                     status = PatchValue.of(from.metadata().status().orEmpty()),
```

Comment:
this sounds fishy with the orEmpty... shouldn't it be just `PatchValue.of(from.metadata().status())` ?

or like in other cases (above in this file)
```
if (from.metadata().hasStatus()) {
    PatchValue.of(from.metadata().status())
} else {
    null
}
```

Please review this and make sure that either `null` is passed to the field if the field was not updated, or `PatchValue.of(<someValue> or <null>)` if the field was updated to someValue or null

Reply:
Good catch, I replaced it with `PatchValue.of(<someValue> or <null>)`. 

However, the final result should match the `if-else` statement you provided. The biggest problem is there's no mechanism like `PatchValue` or `PubNubMembershipMetadataChangeset` for Memberships in the Swift SDK. I think it would be a good idea to consider updating the Swift SDK (including its Objective-C layer for KMP) with a similar mechanism to the one you introduced in Kotlin (PatchValue). All entities, such as User, Channel, and Membership, should be taken into account